client/my-sites/comments/main.jsx

```diff
@@ -47,12 +52,33 @@ export class CommentsManagement extends Component {
 			translate,
 		} = this.props;
 
+		const jetpackUpdateLink = `/plugins/jetpack/${ this.props.siteSlug }`;
+
+		const showJetpackUpdateScreen =
+			isJetpack &&
+			! jetpackCommentsManagementSupported &&
```

Comment:
Is this bang intentional, wouldn't we be checking that it's `true`?

Reply:
Yes, the intention is to show the update screen for Jetpack sites on which comments management is _not_ supported (in this case `<5.5`). Also it worked as expected while I was testing it.